inference-engine/thirdparty/clDNN/tests/test_cases/fusings_gpu_test.cpp

```diff
@@ -3579,3 +3579,115 @@ INSTANTIATE_TEST_CASE_P(fusings_gpu, deconv_scale_actv_quant_u8_eltw_scale_actv_
         deconv_test_params{ CASE_DECONV_S8S8_3D_7, 2, 9 },
         deconv_test_params{ CASE_DECONV_S8S8_3D_8, 2, 9 },
 }), );
+
+/* ----------------------------------------------------------------------------------------------------- */
+/* ---------------------------------------- Eltwise cases -------------------------------------------------- */
+/* ----------------------------------------------------------------------------------------------------- */
+struct eltwise_test_params {
+    tensor input_size;
+    data_types input_type;
+    format input_format;
+    data_types default_type;
+    format default_format;
+    eltwise_mode mode;
+    size_t expected_fused_primitives;
+    size_t expected_not_fused_primitives;
+};
+
+#define CASE_ELTWISE_FP32_1 {2, 16, 4, 4}, data_types::f32, format::bfyx,           data_types::f32,  format::bfyx
+#define CASE_ELTWISE_FP32_2 {2, 16, 4, 4}, data_types::f32, format::bfzyx,          data_types::f32,  format::bfzyx
+#define CASE_ELTWISE_FP32_3 {2, 16, 4, 4}, data_types::f32, format::b_fs_yx_fsv16,  data_types::f32,  format::b_fs_yx_fsv16
+#define CASE_ELTWISE_I8_1   {2, 16, 4, 4}, data_types::i8,  format::bfyx,           data_types::f32,  format::bfyx
+#define CASE_ELTWISE_I8_2   {2, 16, 4, 4}, data_types::i8,  format::bfzyx,          data_types::f32,  format::bfzyx
```

Comment:
Why no fsv16 format for I8 and U8 cases? This format is crucial for low precision cases as well, even if it will be only supported by reference kernel.